src/components/Steps.js

```diff
@@ -175,7 +175,12 @@ export default class Steps extends React.Component {
    * if `opts.ignoreValidation` is true.
    */
   goToStep = async (step, opts = {}) => {
-    if (!opts.ignoreValidation && !(await this.validateCurrentStep())) {
+    const currentStep = this.getStepByName(this.props.currentStepName);
+    if (currentStep.isLastStep && step.index < currentStep.index) {
+      opts.action = 'prev';
```

Comment:
Why only for last step? Going back from 2nd step to first step is going `prev` too

Reply:
I focused on the payment step which I assumed was the `lastStep` in every case. I didn't want other step validation behaviors to be altered.

Reply:
It's a new parameter so others `validate` functions should be safe. From an API consumer point of view, one should expect that `action` is always `prev` when going back, regardless of whether this is the last step or not. Remember that `Steps` component is meant to be generic, so we should treat it as if it were an open source NPM package meant for public use :slightly_smiling_face: